cmd/syncthing/gui.go

```diff
@@ -270,10 +269,12 @@ func (s *apiService) Serve() {
 	postRestMux.HandleFunc("/rest/system/restart", s.postSystemRestart)        // -
 	postRestMux.HandleFunc("/rest/system/shutdown", s.postSystemShutdown)      // -
 	postRestMux.HandleFunc("/rest/system/upgrade", s.postSystemUpgrade)        // -
-	postRestMux.HandleFunc("/rest/system/pause", s.postSystemPause)            // device
-	postRestMux.HandleFunc("/rest/system/resume", s.postSystemResume)          // device
 	postRestMux.HandleFunc("/rest/system/debug", s.postSystemDebug)            // [enable] [disable]
 
+	// Deprecated
+	postRestMux.HandleFunc("/rest/system/pause", s.deprecatedMakePauseHandler(true))   // device
```

Comment:
Sure, but our API for most of the things is posting a config as a whole, hence putting a sugar coating on a pretty shit api still makes it a shit api.

I guess we can add these, hide the option from the config, but then we'd have to query each folders pause state upon refresh, and for a guy with 209 folders that will probably be a bit painful.

Having a monolithic one endpoint API makes things really simple however, 0 extra calls as the config is already there.